apps/docs/src/app/platform/component-docs/platform-split-menu-button/platform-split-menu-button-header/platform-split-menu-button-header.component.html

```diff
@@ -1,9 +1,15 @@
 <header>Split Menu Button</header>
 <description>
-    Split Menu Button has a primary button and a Menu icon. On click on menu dropdown, menu appears. Primary button
-    click the default in Menu.
+    Split Menu Button features a button with a label and an expand icon. The label specifies the default action and the
+    expand icon opens a menu that list the available actions. The split menu button provides a quick and compact way for
+    users to initiate a default action, or choose another action.
+    <h3>Platform specific enhancements:</h3>
+    <ul>
+        <li>Separated creation of Menu reference and Split menu button.</li>
+        <li>Integration of Menu with use of fdpMenuTriggerFor directive.</li>
```

Comment:
When you make this statements, nobody knows what fdMenuTriggerFor is. Unless you reference it or describe more about it I am not sure if this has any value for and app. developer.

Reply:
provided reference for MenuModule where fdMenuTriggerFor is defined.